build/benchmarks.yml

```diff
@@ -120,89 +24,3 @@ jobs:
       warmup: 5
       duration: 15
       connection: 'ASP.NET Benchmarks Queue'
-
-- job: SignalR
```

Comment:
Any particular reason for removing the SignalR job from benchmarks.yml as part of this PR?

Reply:
temp debugging